modules/bbp-morphology/src/main/resources/schemas/bbp/morphology/reconstructedcell/v0.1.0.json

```diff
@@ -0,0 +1,158 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    "{{base}}/contexts/bbp/core/entity/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "bbptlotsh": "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0/shapes/",
+      "bbpentsh": "{{base}}/schemas/bbp/core/entity/v0.1.0/shapes/",
+      "bbplocsh": "{{base}}/schemas/bbp/core/brainlocation/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/bbp/experiment/reconstructedcell/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "owl:Ontology",
+  "imports": [
+    "{{base}}/schemas/bbp/core/entity/v0.1.0",
+    "{{base}}/schemas/bbp/core/typedlabeledontologyterm/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id" : "this:ReconstructedCellShape",
+      "@type" : "sh:NodeShape",
+      "label" : "Reconstructed cell shape definition",
+      "targetClass" : "nsg:ReconstructedCell",
+      "nodeKind" : "sh:BlankNodeOrIRI",
+      "and":[
+        {
+          "node":"bbpentsh:EntityShape"
+        },
+        {
+          "property" : [
+            {
+              "path" : "nsg:projectName",
+              "name" : "Project name",
+              "description" : "Project name as given by the organization who generates the data",
+              "datatype": "xsd:string",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:axonProjection",
+              "name" : "Axon projection",
+              "description" : "Axon projection of the reconstructed cell.",
+              "node":"bbptlotsh:BrainRegionOntologyTermShape",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:brainRegion",
+              "name" : "Brain region",
+              "description" : "Brain region where the reconstructed cell is located.",
+              "node":"bbptlotsh:BrainRegionOntologyTermShape",
+              "minCount" : 1,
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:distanceToBoundary",
+              "name" : "Distance to boundary",
+              "description" : "Distance of the reconstructed cell to pia or to another brain boundary.",
+              "node" : "bbplocsh:DistanceToBoundaryShape",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:mType",
+              "name" : "Morphology type",
+              "description" : "The morphology type of the reconstructed cell: need the cell type for class restriction.",
```

Comment:
I suggest to remove ": need the cell type for class restriction" and to use editorialNote property for editorial comments.